### PR TITLE
Plr student model rework

### PR DESCRIPTION
**Pull request by Caps1d**
Moved the student model file (which is not really student) and the sql file to ../partials/plr as this is where its associated components are located in and thus we avoid having to duplicate code for both plrStaff and plrStudent.

### CODE REVIEW

**Comment by Caps1d** [comment]
main repo pr: https://github.com/UBCO-COSC-499-Summer-2023/PrairieLearn-Gamification/pull/303

File: apps/prairielearn/src/pages/plrStudent/plrStudent.js
Comment (LouisLP) [inline apps/prairielearn/src/pages/plrStudent/plrStudent.js:1]: Delete lines, don't comment them out

**Comment by LouisLP** [comment]
@Caps1d Make sure you pull the recent changes, or branch off the existing branch. Before I merged, it listed all #19 's changes as changes to this branch as well.

**Review by LouisLP — APPROVED** [review]
okay

**Pull request merged**